txircd/module_interface.py

```diff
@@ -108,6 +108,13 @@ def fullUnload():
 		handled by this module.
 		"""
 
+	def verifyConfig(config):
+		"""
+		Called when the module is loaded and when the server is rehashed.  Should check all
+		configuration values defined by this module and either adjust them in the configuration
+		or raise a ConfigError when an invalid value is defined.
```

Comment:
`ConfigValidationError` is probably the only `ConfigError` that should be coming out of these.